### PR TITLE
PYSVC-324 Supported OSs and JVMs

### PR DESCRIPTION
**Pull request by michaelranaldo**

### CODE REVIEW

File: general-info/supported-platforms.adoc [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+== Supported JVMs
+
+Payara Server currently supports the following JVMs:
+
+* Oracle JDK
+* Azul Zulu
+* OpenJDK
+* IBM J9
+
+== Supported OSes
```
Comment (OndrejM) [inline general-info/supported-platforms.adoc]: I think that "Supported Operating Systems" seems better.

File: general-info/supported-platforms.adoc [hunk 2]
```diff
@@ -0,0 +1,35 @@
+= Supported Platforms
+
+Though it is always possible for new versions to be supported, this page
+details the JVMs currently supported with Payara Server.
+
+While not all environments are tested, if it is on a supported JVM we will
+provide support for customers and address issues reported by the community.
+Similarly, other platforms can always be requested for support.
+
+== Supported JVMs
```
Comment (OndrejM) [inline general-info/supported-platforms.adoc]: Maybe  "Supported Java Virtual Machines" is better?

**Comment by OndrejM** [comment]
I think it's OK like this, I would merge it.
@smillidge, what do you think?

**Comment by smillidge** [comment]
Yep fine

**Pull request merged**